modules/specialized-guides/pages/public-cloud-guide/payg/aws/payg-aws-configure-the-image.adoc

```diff
@@ -49,11 +89,13 @@ For more information see: xref:public-cloud-guide/payg/aws/payg-aws-requirements
 
 . You will be redirected to the **EC2** Dashboard.
 
-. Verify that the instance has successfully launched by selecting menu:Instances[] on the left sidebar. The {productname} {payg} instance should be running in the main content area.
+. Verify that the instance has successfully launched by selecting menu:Instances[] on the left sidebar.
+  The {productname} {payg} instance should be running in the main content area.
 
 [NOTE] 
 .Usage and Costs
 ====
-Keep in mind that since this is a {payg} image, you will be billed according to your actual usage, including the number of systems you **manage** and **monitor** with this instance. It's essential to regularly track and review your usage to prevent unexpected costs and ensure alignment with your needs.
+Keep in mind that since this is a {payg} image, you will be billed according to your actual usage, including the number of systems you **manage** and **monitor** with this instance.
```

Comment:
```suggestion
Because this is a {payg} image, you will be billed according to your actual usage, including the number of systems you **manage** and **monitor** with this instance.
```